browser/gemini/gemini_protocol_handler.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_GEMINI_GEMINI_PROTOCOL_HANDLER_H_
+#define BRAVE_BROWSER_GEMINI_GEMINI_PROTOCOL_HANDLER_H_
+
+#include <string>
+
+#include "chrome/browser/external_protocol/external_protocol_handler.h"
```

Comment:
missing includes for WebContents, PageTransision, GURL and base::Optional and external_protocol_handler.h is not needed